### ISSUE TITLE
IIIF Manifest not linked

### ISSUE THREAD

**Issue by klokan** [tomkr/embedr#98]
There is no link to IIIF Manifest at all in the viewer HTML - not even in the "Embed this image" block.
It has been removed.

If anybody want's to use the IIIF tools such as mirador, he needs this. Now he has no way to know there is in fact IIIF Presentation API behind Embedr service.

It has been present in original viewer - removed by @tomkr on the pull requests.

To try IIIF viewer you can test Mirador at http://projectmirador.org/demo/.
- Click "Change layout" to 1x3:
- Click on "+" and add url to manifest...

**Comment by mzeinstra** [comment]
Good catch, we do want this. I'll put it high on the priority of our next developments.
